src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.html

```diff
@@ -33,6 +33,12 @@
         *appFeatureFlag="'show_share_modal'">
         <mat-tab label="ACTIVE">
           <mat-card-content>
+            <mat-slide-toggle
+              class="slide-toggle-my-scenarios"
+              [(ngModel)]="showOnlyMyScenarios"
+              (change)="fetchScenarios()">
+              Show only my scenarios
+            </mat-slide-toggle>
```

Comment:
per chats with UX, we thought this would be an alternative to trying to hack this into the tab row. But this means we copy it twice. Any thoughts on DRYing this? Or should we bother?

Reply:
Maybe it can be part of app-scenarios-table-list? if not, I don't think we should bother